### ISSUE TITLE
Yii2 & Kartik ActiveForm, False positive?

### ISSUE THREAD

**Issue by craiglondon** [kalessil/phpinspectionsea#884]
I am using Yii 2.0.15.1 and Kartik ActiveForm 3.40 (most current for both), this detection seems to be new, as in the past few weeks.

`use kartik\widgets\ActiveForm;`

`Expected kartik\form\string got string` is the text displayed when I hover over 'FullName', 'Address', etc.

```
<div class="row">
    <div class="col-md-6">
        <?php
        echo $form->field($model, 'FullName', ['inputOptions' => ['maxlength' => 50]]);
        echo $form->field($model, 'Address');
        echo $form->field($model, 'City');
        ?>
    </div>
    <div class="col-md-6">
        <?php
        echo $form->field($model, 'State')->dropDownList(Yii::$app->params['states'], ['prompt' => '-- Select State --',]);
        echo $form->field($model, 'Zip', ['inputOptions' => ['minlength' => 5, 'maxlength' => 10]]);  ### Zip Code could also be a Canadian Postal code
        echo $form->field($model, 'Phone')->widget(MaskedInput::className(), ['mask' => '(999) 999-9999', 'options' => ['type' => 'tel', 'class' => 'form-control']]);
        ?>
    </div>
</div>
```

![typecompatissue](https://user-images.githubusercontent.com/3833140/38841566-064f9fea-41b3-11e8-9cf8-b6ef21dc2ba6.jpg)

**Comment by kalessil** [comment]
Hi Craig, this specific inspection is by IDE itself - hence we can not help here.

**Comment by craiglondon** [comment]
Got it.  I forgot that in the last release or two of PHPStorm this plugin has it's on dedicated section in that view.   I'll recreate this issue with them.
